### PR TITLE
DB-10 --> Master

### PR DESCRIPTION
**Pull request by MiguelFieira**
<h1> Changes DB-10</h1>

- Removed Unnecessary Code
- Nested all Css
- Merged DB-1 & DB9 -> DB-10
- Added Grid Layout to Header

<h3> DB-1 Navigation</h3>
- Added Navigation<br>
- Added Icons<br>
- Added Layout<br>
- Added Active Effect<br>
- Added Hover Effect<br>
- Added Dot Under Active Tab<br>

<h3> DB-9 Header</h3>
- Added Company Name<br>
- Added Profile Picture<br>
- Added Logged In Username<br>
- Added Layout<br>

### CODE REVIEW

File: Default/public/index.html [hunk 1]
```diff
@@ -1,32 +1,76 @@
...
+        <div class="notif__text">
+            <h3 class="notif__text--h3">Congratulation Jennie,</h3>
+            <p class="notif__text--p">You've got 208 sales today.<br/> You've reached 6th milestone. checkout<br/> your elite author section</p>
+        </div><br/>
+        <button class="notif__button">View Details</button>
+    </div>
+-->
+</div>
 </body>
-</html>
```
Comment (MatthijsHoek) [inline Default/public/index.html]: remove one </html>

File: Default/public/stylesheets/sass/style.scss [hunk 2]
```diff
@@ -1,96 +1,219 @@
 $font-stack:    Helvetica, sans-serif;
-$primary-color: #333;
```
Comment (MatthijsHoek) [inline Default/public/stylesheets/sass/style.scss]: use one $primary-color

File: Default/public/stylesheets/sass/style.scss [hunk 3]
```diff
@@ -1,96 +1,219 @@
...
+$primary-color: #1875f0;
 $font: 'Roboto', sans-serif;
 
 body {
-  background-color: #7c7c7c;
```
Comment (MatthijsHoek) [inline Default/public/stylesheets/sass/style.scss]: use one background-color

File: Default/public/index.html [hunk 4]
```diff
@@ -1,32 +1,76 @@
...
-    <div class="page-wrapper">
-        <h1>Groentemix</h1>
 
-        <div class="visitors">
-            <h4 class="visitors__header">Views</h4>
-            <p class="visitors__total">6080 sessions</p>
-            <button class="visitors__button">More Details</button>
-            <img src="img/visitors.PNG" class="visitors__graph">
+<body>
+<div class="page-wrapper">
```
[the remark this answers is not in the document]
Comment (MiguelFieira) [reply Default/public/index.html]: Fixed

File: Default/public/stylesheets/sass/style.scss [hunk 5]
```diff
@@ -1,96 +1,219 @@
...
+  background-color: #f2f2f2;
```
[the remark this answers is not in the document]
Comment (MiguelFieira) [reply Default/public/stylesheets/sass/style.scss]: fixed

**Pull request merged**